templates/fine-tune-llm_v2/end-to-end-examples/fine-tune-function-calling/README.ipynb

```diff
@@ -529,13 +356,12 @@
    "metadata": {},
    "source": [
     "For fine-tuning, you have two options with Anyscale:\n",
-    "1. Fine-tuning on the Anyscale Platform through our fine-tuning template \n",
+    "1. Fine-tuning on the Anyscale Platform through our [fine-tuning template](../../README.md)\n",
```

Comment:
How will all of these relative links pop up on the docs ? 

Maybe that question is left to when we figure out how to surface cookbooks/end-to-end examples in the docs? Looks like at that point we might have to decouple doc generation from the .ipynb notebook as well.  (because it's nice to have this relative link on platform, but not in my .md file that will be on the docs page)


I am okay with keeping this here for the platform exp.

Reply:
This is sth for the docs team to figure out. I will run the docs page update and if it didn't look right will have to report to them